tests/integration/components/filter-menu-test.js

```diff
@@ -0,0 +1,18 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { findAll, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | filter-menu', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test(`it renders and it's not empty`, async function(assert) {
+    await render(hbs`<FilterMenu />`);
+    assert.notEqual(this.element.textContent.trim(), '');
+  });
+
```

Comment:
In math, we can think of the 1st test as a **weak statement**, and the 2nd test as a **strong statement**. A strong statement always implies a weak statement (assuming the statements are related).

In the context of `<FilterMenu>` component, if there is more than 1 link (what the 2nd test checks), then we know that the component is not empty (the 1st test).

We can use the idea of strong and and weak statements to remove tests and assertions that aren't necessary to write and check. Decrease time to run tests. ✨

```suggestion
```